src/Controls/src/Core/FlyoutBase.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace Microsoft.Maui.Controls
+{
+	public abstract class FlyoutBase : Element, IFlyout
+	{
+		public static readonly BindableProperty ContextFlyoutProperty = BindableProperty.CreateAttached("ContextFlyout", typeof(FlyoutBase), typeof(FlyoutBase), null,
+			propertyChanged: (bo, oldV, newV) =>
+			{
+				if (oldV is BindableObject oldMenu)
+					VisualElement.SetInheritedBindingContext(oldMenu, null);
+
+				if (newV is BindableObject newMenu)
+					VisualElement.SetInheritedBindingContext(newMenu, bo.BindingContext);
+
+			});
```

Comment:
Will this still work when I change a view's binding context? Does this `SetInheritedBindingContext` set up events to listen for BC changes to the parent?